engine/src/main/java/org/terasology/logic/players/WorldtimeResyncSystem.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Please use the new license header at least in completely new files (ideally update in existing files if not updated yet).
```
// Copyright 2020 The Terasology Foundation
// SPDX-License-Identifier: Apache-2.0
```

Reply:
The license header is now corrected in the 6 files changed of this PR.